libs/books/feature/src/lib/reading-list/reading-list.component.spec.ts

```diff
@@ -1,26 +1,53 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 import { SharedTestingModule } from '@tmo/shared/testing';
+import { MatSnackBarHarness } from '@angular/material/snack-bar/testing'
+import { TestbedHarnessEnvironment } from '@angular/cdk/testing/testbed'
+import { HarnessLoader } from '@angular/cdk/testing';
 
 import { ReadingListComponent } from './reading-list.component';
 import { BooksFeatureModule } from '@tmo/books/feature';
+import { NoopAnimationsModule } from '@angular/platform-browser/animations';
+import { MatSnackBarModule } from '@angular/material/snack-bar';
 
 describe('ReadingListComponent', () => {
   let component: ReadingListComponent;
   let fixture: ComponentFixture<ReadingListComponent>;
+  let loader: HarnessLoader;
+  const options = {      
+    message: "Book is being removed from Reading List",
+    action: "Undo",
+    options: {
+      duration: 5000,
+      verticalPosition: 'top'
+    }
+  };
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      imports: [BooksFeatureModule, SharedTestingModule]
+      imports: [BooksFeatureModule, MatSnackBarModule, NoopAnimationsModule, SharedTestingModule],
     }).compileComponents();
   }));
 
   beforeEach(() => {
     fixture = TestBed.createComponent(ReadingListComponent);
     component = fixture.componentInstance;
     fixture.detectChanges();
+    loader = TestbedHarnessEnvironment.documentRootLoader(fixture);
   });
 
   it('should create', () => {
-    expect(component).toBeTruthy();
+    expect(component).toBeDefined();
+  });
+
+  it('should load harness for snack-bar', async () => {
```

Comment:
Divide the test into two, for load and dismiss snackbar. Keep only one action per test case.